projects/packages/forms/src/dashboard/admin-migrate-page/index.tsx

```diff
@@ -38,6 +40,15 @@ const AdminMigratePage = () => {
 			</span>
 		</div>
 	);
+
+	const onCheckNewFormsClick = useCallback( () => {
+		jetpackAnalytics.tracks.recordEvent( 'jetpack_wpa_forms_new_forms_page_cta_click', {
```

Comment:
why "wpa" here?

Reply:
Good catch. I copied from another event and only edited the suffix. Removing it.